templates/docs/examples/brochure/_25-75-nested.html

```diff
@@ -1,6 +1,6 @@
 <section class="p-section">
   <div class="row">
-    <hr class="p-rule">
```

Comment:
As this is showcased as the rule component example, please bring `p-rule` class names back in this file.

Reply:
I think we need this `.p-rule` class reintroduced into this template, as Bartek suggested?